flink-connectors/flink-connector-kinesis/src/main/resources/META-INF/NOTICE

```diff
@@ -8,47 +8,47 @@ This project bundles the following dependencies under the Apache Software Licens
 
 - com.amazonaws:amazon-kinesis-client:1.11.2
 - com.amazonaws:amazon-kinesis-producer:0.14.0
-- com.amazonaws:aws-java-sdk-core:1.11.788
+- com.amazonaws:aws-java-sdk-core:1.11.754
```

Comment:
ah...I guess we actually changed this in #13758...

Reply:
Yes, that was a mistake